### ISSUE TITLE
Unicode Character 'LINE SEPARATOR' (U+2028) not displayed correctly

### ISSUE THREAD

**Issue by apblack** [gracelang/grace-web-editor#33]
Grace recognizes Unicode Character LINE SEPARATOR (U+2028) as a valid way of ending a line of code, along with LINE FEED (U+000A) and 'CARRIAGE RETURN  (U+000D).   But the editor (i.e., Ace) displays it as a centered dot, not as a line break.  (The compiler accepts it just fine).

Can the editor break the line when it sees a LINE SEPARATOR character?

**Comment by Dmitri-2** [comment]
I've been studying this issue, and was wondering whether a different fix would be acceptable. Could the editor instead simply replace the unicode character with a newline when it sees it? To fix the issue in uploaded files, I could also replace all of the unicode Line Separator characters with newlines before they get to the editor.

**Comment by apblack** [comment]
I don't think that changing the contents of the user's file is acceptable.  I've posted this as an issue for ACE, but don't have much hope that they will fix it.  Let's just leave it open here, but not do anything about it.
